version 1.9

EvgeniyKuch opened the pull request:
##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB) 
Добавлена обёркта приложения в docker-контейнер с помощью jib-maven-plugin. 
Во время mvn package происходит создание образа непосредственно в демоне Docker.

Commit:
version 1.9

##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB)
Добавлена обёркта приложения в docker-контейнер с помощью jib-maven-plugin.
Во время mvn package происходит создание образа непосредственно в демоне Docker.

Commit:
version 2.0

Добавлена обёркта приложения в docker-контейнер с помощью jib-maven-plugin.
Во время mvn package происходит создание образа непосредственно в демоне Docker.

Добавлен docker-compose.yml для связи с MongoDB.
Старт docker-compose up.

kataus commented on library-spring-data-mongo-mvc-auth-url/src/main/java/ru/otus/library/config/MongoConfig.java:
```diff
@@ -19,7 +19,7 @@ protected String getDatabaseName() {
     @Override
     public MongoClient mongoClient() {
-        final ConnectionString connectionString = new ConnectionString("mongodb://localhost:27017/library");
+        final ConnectionString connectionString = new ConnectionString("mongodb://mongo:27017/library");
```
Лучше вынести в настройки, чтобы из docker-compose получать. Мало ли что понадобится

kataus commented on library-spring-data-mongo-mvc-auth-url/docker-compose.yml:
```diff
@@ -0,0 +1,16 @@
+version: '1'
+services:
+  book-library:
+    image: "book-library"
```
стоит использовать параметр depends_on, потому что без базы приложение в общем нам не нужно

Commit:
version 2.1

##### Приложение, хранящее информацию о книгах в библиотеке (на MongoDB)
Добавлена обёркта приложения в docker-контейнер с помощью jib-maven-plugin.
Во время mvn package происходит создание образа непосредственно в демоне Docker.

Добавлен docker-compose.yml для связи с MongoDB, которая обёрнута в отдельный контейнер.
Старт docker-compose up.

Параметры подключения к Mongo вынесены в настройки и переопределны в docker-compose.yml.
В docker-compose.yml добавлен depends_on, чтобы база стартовала раньше.
